Testscripts/Linux/validate-intel-sgx-driver.sh

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+#######################################################################
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+#
+# validate-intel-sgx-driver.sh
+# Description:
+#    Check the existence of Intel sgx driver.
+#    Install Open-Enclave and build&run Samples
+# Supported Distros:
+#    Ubuntu
+#######################################################################
+
+set -e
+set -x
+
+. utils.sh || {
+    echo "Error: unable to source utils.sh!"
+    echo "TestAborted" >state.txt
+    exit 1
+}
+
+UtilsInit
+
+sudo apt-get update
+sudo apt-get -y upgrade
+echo 'deb [arch=amd64] https://download.01.org/intel-sgx/sgx_repo/ubuntu bionic main' | sudo tee /etc/apt/sources.list.d/intel-sgx.list
+wget -qO - https://download.01.org/intel-sgx/sgx_repo/ubuntu/intel-sgx-deb.key | sudo apt-key add -
+
+echo "deb http://apt.llvm.org/bionic/ llvm-toolchain-bionic-7 main" | sudo tee /etc/apt/sources.list.d/llvm-toolchain-bionic-7.list
```

Comment:
Logging will help in the failure analysis. Please add logging msg if you think helpful.

Reply:
-x option is enabled

Reply:
If I remember correct, that is printing on console or terminal. My reference is LogMsg for test automation log. It will be used for future analysis.

Reply:
All the commands and output will be printed to "VALIDATE-INTEL-SGX-DRIVER.log" and then downloaded to LogDir/.

Reply:
VALIDATE-INTEL-SGX-DRIVER.log will include all the commands and output.

Reply:
Acked, Your test case has its own test log. I fine with it then.